src/Microsoft.Health.Fhir.Api/Features/ActionResults/BulkImportResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
```

Comment:
fhir-server\src\Microsoft.Health.Fhir.Shared.Web\roles.json is missing

Reply:
Added